code/game/objects/items/sticker/misc.dm

```diff
@@ -0,0 +1,72 @@
+/*
+	Series 1
+*/
+
+/obj/item/sticker/series_1/get_stats()
+	. = "<span class='notice'>Series 1</span>\n"
+	. += ..()
+
+/obj/item/sticker/series_1/generate_unusual()
+	var/obj/emitter/emitter = pick(list(/obj/emitter/electrified, /obj/emitter/snow, /obj/emitter/fire))
+	if(prob(1))
+		playsound(src, 'sound/effects/audience-gasp.ogg', 50)
+		add_emitter(emitter, "unusual", 10)
+		is_unusual = TRUE
+
+/obj/item/sticker/series_1/smile
+	sticker_flags = STICKER_SERIES_1 | STICKER_RARITY_COMMON
+
+/obj/item/sticker/series_1/skub
+	icon_state = "skub"
+	sticker_icon_state = "skub_sticker"
+	sticker_flags = STICKER_SERIES_1 | STICKER_RARITY_MYTHIC
+	//Don't chance the drop weight for this, the joke is it's common
+
+/obj/item/sticker/series_1/c4
+	icon_state = "c4"
+	sticker_icon_state = "c4_sticker"
+	do_outline = FALSE
+	sticker_flags = STICKER_SERIES_1 | STICKER_RARITY_RARE
+	drop_rate = STICKER_WEIGHT_RARE
+
+/obj/item/sticker/series_1/sad
+	icon_state = "sad"
+	sticker_icon_state = "sad_sticker"
+	sticker_flags = STICKER_SERIES_1 | STICKER_RARITY_COMMON
+
+/obj/item/sticker/series_1/moth
+	icon_state = "moth"
+	sticker_icon_state = "moth_sticker"
+	sticker_flags = STICKER_SERIES_1 | STICKER_RARITY_UNCOMMON
+	drop_rate = STICKER_WEIGHT_UNCOMMON
+
+/obj/item/sticker/series_1/pride
+	item_flags = ABSTRACT
+	sticker_flags = STICKER_SERIES_1 | STICKER_RARITY_UNCOMMON
+	drop_rate = STICKER_WEIGHT_UNCOMMON
+
+/obj/item/sticker/series_1/pride/gay
+	item_flags = NONE
+	icon_state = "gay"
+	sticker_icon_state = "gay_sticker"
+
+/obj/item/sticker/series_1/pride/lesbian
+	item_flags = NONE
+	icon_state = "lesbian"
+	sticker_icon_state = "lesbian_sticker"
+
+/obj/item/sticker/series_1/pride/bi
+	item_flags = NONE
+	icon_state = "bi"
+	sticker_icon_state = "bi_sticker"
+
+/obj/item/sticker/series_1/pride/trans
+	item_flags = NONE
+	icon_state = "trans"
+	sticker_icon_state = "trans_sticker"
+
+//Mime pride?
+/obj/item/sticker/series_1/pride/straight
+	item_flags = NONE
+	icon_state = "straight"
+	sticker_icon_state = "straight_sticker"
```

Comment:
I think we already have a few similar references in the game, which includes both real and pride flags.
I don't really see too much of a problem with it, and I'd personally like to keep them in, if it's negotiable.

Reply:
Since it can be quite stressful to draw a line between real-world issues and the fictional universe and this is an offspring mechanic unrelated to the core artifact PR, I'd prefer if in this scenario we just maintain a clear separation between the IC content and any real-world issues entirely. I don't see this stance here as a permanent precedence and I just want to be safe here since the PR is already extremely difficult to review given the nature of artifacts, so this isn't something that will necessarilly apply to other PRs/features going forward.